wasm/build_all.bat

```diff
@@ -0,0 +1,13 @@
+node wajicup.js -cc -std=c11 ../source/dos.c dos.o
```

Comment:
Just a bit curious, is there anything in particular which required C11?

Reply:
Without the std=c11 clang prints these 4 warnings:
```c
In file included from ../source/dos.c:5:
In file included from ../source/dos.h:3465:
../source/libs/opblib.h:330:3: warning: redefinition of typedef 'Context' is a C11 feature [-Wtypedef-redefinition]
} Context;
  ^
../source/libs/opblib.h:311:24: note: previous definition is here
typedef struct Context Context;
                       ^
../source/libs/opblib.h:433:3: warning: redefinition of typedef 'Command' is a C11 feature [-Wtypedef-redefinition]
} Command;
  ^
../source/libs/opblib.h:312:24: note: previous definition is here
typedef struct Command Command;
                       ^
../source/libs/opblib.h:438:3: warning: redefinition of typedef 'OpbData' is a C11 feature [-Wtypedef-redefinition]
} OpbData;
  ^
../source/libs/opblib.h:313:24: note: previous definition is here
typedef struct OpbData OpbData;
                       ^
../source/libs/opblib.h:545:3: warning: redefinition of typedef 'Instrument' is a C11 feature [-Wtypedef-redefinition]
} Instrument;
  ^
../source/libs/opblib.h:314:27: note: previous definition is here
typedef struct Instrument Instrument;
                          ^
```

Also getting opblib.h to build with MSVC 2013 was a bit tedious, I had to convert the `0b10000000` notion to `0x80`. I'm surprised tcc does it all so well.